widget/src/index.tsx

```diff
@@ -31,34 +31,42 @@ export const ElementSelector = {
   PurchaseButtonClassName: "ab-wpr-purchase-btn"
 };
 
-ReactDOM.render(
-  <Provider inject={[appState, appState.state.userSession]}>
-    <AppRoot/>
-  </Provider>
-  , document.getElementById(ElementSelector.AppRootId));
+if (document.getElementById(ElementSelector.AppRootId)) {
```

Comment:
This element is mandatory to be exist, please also print an error message to the console if this element doesn't exist.